client/my-sites/migrate/section-migrate.jsx

```diff
@@ -287,59 +287,69 @@ class SectionMigrate extends Component {
 	}
 
 	renderMigrationComplete() {
-		const { targetSite } = this.props;
+		const { targetSite, translate } = this.props;
 		const viewSiteURL = get( targetSite, 'URL' );
 
 		return (
 			<>
 				<FormattedHeader
 					className="migrate__section-header"
-					headerText="Congratulations!"
+					headerText={ translate( 'Congratulations!' ) }
 					align="left"
 				/>
 				<CompactCard>
-					<div className="migrate__status">Your import has completed successfully.</div>
+					<div className="migrate__status">
+						{ translate( 'Your import has completed successfully.' ) }
+					</div>
 					<Button primary href={ viewSiteURL }>
-						View site
+						{ translate( 'View site' ) }
 					</Button>
 					<Button onClick={ this.resetMigration }>Start over</Button>
 				</CompactCard>
 			</>
 		);
 	}
 
-	renderMigrationConfirmation() {}
-
 	renderMigrationError() {
+		const { translate } = this.props;
+
 		return (
 			<Card className="migrate__pane">
 				<FormattedHeader
 					className="migrate__section-header"
-					headerText="Import failed"
+					headerText={ translate( 'Import failed' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **42** times:
`translate( 'Import Failed ' )` *ES Score: 9*

Reply:
Would it work to use the suggsted string, given this is a header?

Reply:
Honestly, the suggested string isn't great, and we should probably replace it, wherever it is. I'm not sure why the string ends in a space.

Reply:
Ah well spotted!